.github/workflows/scripts/start-acs.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+set -euox pipefail
+
+
+cd $STACKROX_DIR
+
+echo "GITHUB_OUTPUT= ${GITHUB_OUTPUT}"
+./deploy/k8s/central.sh
+kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
+sleep 20
```

Comment:
I think we should avoid sleeping if possible. Could we check differently (e.g., with `curl` or using `wait-for` script) if the connection is already open?

Additionally, we need to cleanup the port forward after we are done with it. You can get inspiration how to do this from here: https://github.com/stackrox/stackrox/blob/master/tools/local-compliance/scripts/local-compliance.sh

Reply:
These two things were already in the cluster.yml file. I think this is outside of the scope of this PR.